experimentalSetups/ankleSetup/telemetry/telemetry.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE devices PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+
+    <device xmlns:xi="http://www.w3.org/2001/XInclude" name="telemetryDeviceDumper" type="telemetryDeviceDumper">
+        <param name="axesNames">(ankle_joint)</param> <!--shoulder_pitch, shoulder_roll, shoulder_yaw, elbow-->
+        <param name="logIEncoders">true</param>
+        <param name="logITorqueControl">false</param>
+        <param name="logIMotorEncoders">true</param>
+        <param name="logIControlMode">false</param>
+        <param name="logIInteractionMode">false</param>
+        <param name="logIPidControl">false</param>
+        <param name="logIAmplifierControl">true</param>
+        <param name="logIRawValuesPublisher">true</param>
+        <param name="saveBufferManagerConfiguration">true</param>
+        <param name="experimentName">ankleSetup_test</param>
+        <param name="path">./ankleSetup_telemetry_data/</param>
+        <param name="n_samples">800000</param>
+        <param name="save_periodically">false</param>
+        <param name="save_period">1000.0</param>
+        <param name="log_period">0.002</param>
+        <!-- <param name="data_threshold">50000</param> -->
+        <param name="auto_save">true</param>
+        <param name="yarp_robot_name">ergoCubSN002S</param>
+        <param name="rawValuesPublisherRemoteName">/fakedevicetest/raw_data</param>
```

Comment:
please use a more significant name instead of `fakedevicetest`, like `ankleSetupTelemtery` or something else. Be careful to substitute the name also in the ankle-setup-rawval-nws_wrapper.xml file. thanks

Reply:
Done!